docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.text.scss

```diff
@@ -133,23 +134,43 @@
   }
 }
 
-@mixin hb-link--inline {
+// Link style for main content that is not a heading
+@mixin hb-link--inline($icon: false) {
   // no font size by default
   @include hb-colorful {
     @include hb-pairing-color('color', 'tertiary');
+    background-size: 100% 200%;
+    background-position-y: -100%;
+    background-repeat: no-repeat;
+
+    // For links without an icon
```

Comment:
can you also write a comment about what this does for external links specifically? thanks!